install-build-deps.sh

```diff
@@ -4,7 +4,7 @@
 # `make -f Makefile.docker update-build-image-manifest && make -f Makefile.docker push-build-image`
 
 if [ -z "${GOBIN+x}" ]; then
- GOBIN="$(go env GOPATH)/bin"
+ GOBIN="${GOPATH%:*}/bin"
```

Comment:
i think the greedy expansion you had before (with double `%%`) was better since that will support >2 paths